FastSimulation/Tracking/plugins/TrackCandidateProducer.cc

```diff
@@ -85,7 +85,8 @@ TrackCandidateProducer::produce(edm::Event& e, const edm::EventSetup& es) {
 
   edm::ESHandle<Propagator>             propagator;
   es.get<TrackingComponentsRecord>().get(propagatorLabel,propagator);
-    
+  Propagator* thePropagator = propagator.product()->clone();
```

Comment:
don't clone
instead get a propagator that doesn't require change,
by using propagatorLabel "PropagatorWithMaterialOpposite"